GooglePlayInstant/LoadingScreen/LoadingBarGenerator.cs

```diff
@@ -22,20 +22,23 @@ namespace GooglePlayInstant.LoadingScreen
     /// </summary>
     public static class LoadingBarGenerator
     {
-        // Loading bar size as a proportion of the screen size. Adjust if needed.
-        private const float LoadingBarWidthProportion = 0.7f;
-        private const float LoadingBarHeightProportion = 0.02f;
+        // Loading bar size.

```

Comment:
"Loading bar size in pixels." or whatever the units are.

Reply:
Resolved